run/run.py

```diff
@@ -345,6 +358,28 @@ def get_config():
     return config['default']
 
 
+def patch_wpt(config, platform):
+    '''Applies util/wpt.patch to WPT.
+
+    The patch is necessary to keep WPT running on long runs.
+    jeffcarp has a PR out with this patch:
+    https://github.com/w3c/web-platform-tests/pull/5774
+    '''
+    patch_path = '%s/util/wpt.patch' % config['wptd_path']
+    with open(patch_path) as f:
+        patch = f.read()
+
+    # Hackery since the --sauce-platform command line arg doesn't
+    # accept spaces, but Sauce requires them in the platform name.
+    patch = patch.replace('__platform_hack__', '%s %s' % (
+        platform['os_name'], platform['os_version'])
+    )
+
+    cmd = ['git', 'apply', '-']
```

Comment:
This variable is probably not necessary (can be inlined), and is also inconsistent since above you use a variable name `command`.